tests/test_tools.py

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env pytest -vs
 """Tests for Gophish tool functions."""
 
+# Standard Python Libraries
+from unittest.mock import patch
+
 # Third-Party Libraries
-from mock import patch
```

Comment:
Is `mock` being used anywhere after the changes in this PR? It looks like it was rolled into `unittest.mock` in Python 3.3 so I think we should remove it from the testing requirements and remove the typing package from the `mypy` configuration. I'm also unsure if we need the `pytest-mock` package for the similar reasons.

Reply:
`mock` was not being used, so I was able to remove it with https://github.com/cisagov/gophish-tools/pull/123/commits/7a0e4392fb8603b8353563b262deb97029dd2095.

`pytest-mock` is still needed here for the `mocker` fixture:
https://github.com/cisagov/gophish-tools/blob/2ff2234bae95f86f7591cf2cac9b3f33e1554a4e/tests/test_input.py#L21-L23

While I was here, I added some missing type modules (https://github.com/cisagov/gophish-tools/pull/123/commits/a9fe37e7b20dde161347fc835cb674aa88394e7e) and some ignore directives (https://github.com/cisagov/gophish-tools/pull/123/commits/2ff2234bae95f86f7591cf2cac9b3f33e1554a4e) to make `mypy` happy.